streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -86,6 +87,14 @@ private TaskMetrics() {}
     private static final String NUM_BUFFERED_RECORDS_DESCRIPTION = "The count of buffered records that are polled " +
         "from consumer and not yet processed for this active task";
 
+    private static final String RECORD_E2E_LATENCY = "record-e2e-latency";
+    static final String RECORD_E2E_LATENCY_MAX_DESCRIPTION =
```

Comment:
req: Please define this constant as private as all the others. I left a request in `TaskMetricsTest` which makes this request clearer.